substrate/frame/broker/src/tests.rs

```diff
@@ -1092,3 +1093,141 @@ fn config_works() {
 		assert_noop!(Broker::configure(Root.into(), cfg), Error::<Test>::InvalidConfig);
 	});
 }
+
+/// Ensure that a lease that ended before `start_sales` was called can be renewed.
+#[test]
+fn renewal_works_leases_ended_before_start_sales() {
```

Comment:
more like `renewal_works_for_leases_ended_before_period_1()` ?

And description:

Bootstrapping: Ensure that leases that end right in the first sale period can still be renewed, despite the fact that renewal rights are determined one period before the corresponding sale period.

Reply:
No, I mean this lease ends before `start_sales` :P

Reply:
Yeah, but the issue is wider than that. Even leases after `start_sales` would not have renewal rights, without your fix.

Reply:
haha I just called them `short_leases` in mine